### PR TITLE
POC-3 #Add facilities to EID mapping

### PR DESCRIPTION
**Pull request by hiqedme**
New facilities added

### CODE REVIEW

File: service/eid/eid-facility-mappings.json [hunk 1]
```diff
@@ -2501,6 +2501,132 @@
...
+    "description": "This is a module under KCRH",
+    "mflCode": "16335",
+    "county": "EMC",
+    "cityVillage": "",
+    "eidDisplay": ""
+  },
+  "086e6f2e-b67e-4926-a8fa-23999a7b7af1": {
+    "mrsId": 433,
+    "mrsDisplay": "Kaptarakwa Sub County Hospital",
+    "description": "This is a module under KCRH",
```
Comment (drizzentic) [inline service/eid/eid-facility-mappings.json:2624]: For description, we can re-use the mrsDisplay

File: service/eid/eid-facility-mappings.json [hunk 2]
```diff
@@ -2501,6 +2501,132 @@
...
+    "mflCode": "14776",
+    "county": "EMC",
```
Comment (drizzentic) [inline service/eid/eid-facility-mappings.json:2626]: Let's have this in full. Elgeyo Marakwet and transzoia as Transzoia

**Review by drizzentic — CHANGES_REQUESTED** [review]

File: service/eid/eid-facility-mappings.json (hunk 1 above)
Comment (hiqedme) [reply service/eid/eid-facility-mappings.json:2624]: okay
on it

Comment (hiqedme) [reply service/eid/eid-facility-mappings.json]: Done

File: .gitignore [hunk 3]
```diff
@@ -11,3 +11,4 @@ etl-daemon.js
 coverage
 conf/config-bk.json
 .DS_Store
+package-lock.json
```
Comment (drizzentic) [inline .gitignore:14]: We shouldn't ignore this file. It ensures everyone has the same version of dependencies.

Comment (hiqedme) [reply .gitignore:14]: Noted.

**Review by drizzentic — APPROVED** [review]

**Pull request merged**